keps/sig-storage/3751-volume-attributes-class/README.md

```diff
@@ -172,27 +172,49 @@ type PersistentVolumeClaimSpec struct {
   ...
 }
 
+// Add the PersistentVolumeClaimModifyVolume condition to existing PersistentVolumeClaimConditionType
+// Condition is used to document the last error controller sees
+const (
+  ...
+  // Applying the target VolumeAttributesClass encountered an error
+  PersistentVolumeClaimVolumeModifyVolumeError PersistentVolumeClaimConditionType = "ModifyVolumeError"
+  // Volume is being modified
+  PersistentVolumeClaimVolumeModifyingVolume PersistentVolumeClaimConditionType = "ModifyingVolume"
+  ...
+)
+
+
 // PersistentVolumeClaimStatus represents the status of PV claim
 type PersistentVolumeClaimStatus struct {
   ...
-  // The VolumeAttributesClassName string cannot be empty
-  VolumeAttributesClassName string
-  ModifyVolumeStatus *PersistentVolumeClaimModifyVolumeStatus
+  // CurrentVolumeAttributesClassName is the current name of the VolumeAttributesClass the PVC is using
+  CurrentVolumeAttributesClassName *string
+  // ModifyVolumeStatus represents the status object of ControllerModifyVolume operation
+  ModifyVolumeStatus *ModifyVolumeStatus
   ...
 }
 
+// ModifyVolumeStatus represents the status object of ControllerModifyVolume operation
```

Comment:
If we don't want to distinct the ControllerModifyVolume and XXXModifyVolume, Is it possible to drop the Controller prefix in the ModifyVolumeStatus description?

Reply:
It is already dropped in the newest revision?

Reply:
the `ControllerModifyVolume ` word still exists in L197 and L201